packages/dara-components/js/common/table/table.tsx

```diff
@@ -373,12 +373,12 @@ function Table(props: TableProps): JSX.Element {
 
     const datetimeColumns = useMemo(() => getDatetimeColumns(resolvedColumns), [resolvedColumns]);
     const fetchData = useCallback(
-        async (startIndex: number, stopIndex: number) => {
+        async (index?: number) => {
```

Comment:
I think this function still needs to accept the start/end index for the infinite loader to work correctly. I was just referring to the `getRowByIndex` not having to pass in those indices.

Reply:
Oh I see, misunderstood what you meant, fixed it now.